npm run zip-azure-function not working on Windows

**Deployment**  
_My deployment is on:_

- [ ] Docker
- [ ] AWS Lambda
- [ ] GCP
- [x] Azure
- [ ] Other _Please specify:_

**Expected behaviour**

Running `npm run zip-azure-function` generates a zip.

**Actual behaviour**

An error occurs
```
> mollie-commercetools-extension-module@1.0.0 build C:\x\y\z\mollie\commercetools-1.0.0\extension
> node_modules/typescript/bin/tsc

'node_modules' is not recognized as an internal or external command,
operable program or batch file.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! mollie-commercetools-extension-module@1.0.0 build: `node_modules/typescript/bin/tsc`
npm ERR!
npm ERR! Failed at the mollie-commercetools-extension-module@1.0.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

**Attempted fixes**

I fixed this by running the command from a windows subsystem for linux (WSL) terminal.
I had to install node and zip first.

**Please provide more information about how this can be reproduced. Please include jsfiddle links, code snippets or any other necessary information**

OS Name: Microsoft Windows 10 Enterprise (10.0.19043 Build 19043)
Node version: 14.18.3

Follow the instructions as explained here:
![image](https://user-images.githubusercontent.com/44951819/161100808-ca65b950-c709-40ae-847b-7eaa432c2c54.png)

**Advice**

It would be nice if the following additional information could be added to the instructions:

- Operating system on which to run the command. (The command would (never?) work on Windows for example, because the **zip** command does not exist as far as I know.)
- Programs + version someone needs to have installed
   - zip
   - node (I'm guessing latest 14.x?)

--- a/docs/Deployment.md
+++ b/docs/Deployment.md
@@ -18,9 +18,11 @@
       - [Basic](#basic)
   * [Docker](#docker)
 
-## NodeJS Runtime
 
-The recommended runtime version is 14
+## Prerequisites
+
+Commands and scripts in this documentation require bash shell, nodejs with npm, zip and docker.
+The recommended NodeJS runtime version is 14
 
 ## Environment variables
 
@@ -252,7 +254,3 @@ Note that the environment variables that are required should be passed with the 
 
 When finished, to stop the container, run:
 `docker stop ct-mollie-extension`
-
-## Prerequisites
-
-Commands and scripts in this documentation require bash shell, nodejs with npm, zip and docker.

--- a/docs/Deployment.md
+++ b/docs/Deployment.md
@@ -252,3 +252,7 @@ Note that the environment variables that are required should be passed with the 
 
 When finished, to stop the container, run:
 `docker stop ct-mollie-extension`
+
+## Prerequisites
+
+Commands and scripts in this documentation require bash shell, nodejs with npm, zip and docker.

